generate_tuxbuild.py

```diff
@@ -20,31 +20,30 @@ def get_config():
 def get_repo_ref(config, tree_name):
     for tree in config["trees"]:
         if tree["name"] == tree_name:
-            return tree["repo"], tree["ref"]
+            return tree["git_repo"], tree["git_ref"]
 
 
 def emit_tuxbuild_yml(config, tree):
-    repo, ref = get_repo_ref(config, tree)
     print("# DO NOT MODIFY MANUALLY!")
     print("# This file has been autogenerated by invoking:")
     print("# $ ./generate_tuxbuild.py < generator.yml {} > tuxbuild/{}.tux.yml".format(tree, tree))
     print("# Invoke tuxbuild via:")
-    print("# $ tuxbuild build-set --git-repo '{}' --git-ref {} \\".format(repo, ref))
-    print("#     --set-name foo-bar --json-out builds.json --tux-config {}.tux.yml".format(tree))
+    print("# $ tuxbuild build-set --set-name foo-bar --json-out builds.json --tux-config {}.tux.yml".format(tree))
     print("""\
 sets:
   - name: foo-bar
     builds:\
 """)
+    repo, ref = get_repo_ref(config, tree)
     max_version = max(config["llvm_versions"])
     for build in config["builds"]:
-        if build["repo"] == repo and build["ref"] == ref:
+        if build["git_repo"] == repo and build["git_ref"] == ref:
             arch = build["ARCH"] if "ARCH" in build else "x86_64"
             toolchain = "clang-"
             toolchain += "nightly" if build["llvm_version"] == max_version else str(build["llvm_version"])
             kconfig = build["config"]
-            print("      - {{target_arch: {0}, toolchain: {1}, kconfig: {2}, make_variables: {{LLVM: \"{3}\"}}}}".format(\
-                    arch, toolchain, kconfig, int(build["llvm"])))
+            print("      - {{git_repo: \"{0}\", git_ref: \"{1}\", target_arch: {2}, toolchain: {3}, kconfig: {4}, make_variables: {{LLVM: \"{5}\"}}}}".format(\
```

Comment:
Can we avoid the escaped double quotes here? I suspect it's only needed when serializing an `int` as a `str`.

Reply:
Yeah, you can see it's adding `""` to the generated yml. We should be able to leave those off.

Reply:
I am not sure, the documentation for tuxbuild shows them quoted. I did not try though.

Reply:
They no longer need to be quoted for tuxbuild - that was a bug and it was fixed..  Though in general this is easier to do if you create the data structure and then use print(yaml.dump()). Creating yaml (or json) using printed strings is harrowing!